docs/snippets/cpu/compile_model.py

```diff
@@ -3,15 +3,15 @@
 
 
 #! [compile_model_default]
-from openvino.runtime import Core
+import openvino as ov
 
-core = Core()
+core = ov.Core()
 model = core.read_model("model.xml")
-compiled_model = core.compile_model(model, "CPU")
+compiled_model = ov.compile_model(model, "CPU")
```

Comment:
```suggestion
compiled_model = core.compile_model(model, "CPU")
```
Don't mix with `convert_model` :)

Reply:
Hi, so I assume all other methods listed in init:
https://github.com/sgolebiewski-intel/openvino/blob/master/src/bindings/python/src/openvino/__init__.py

are also to be reverted?

@jiwaszki Can you relate?

Reply:
good question. I really forgot that we had `compile_model` without `Core` in Python API.
@slyalin @jiwaszki what is your preference here? and why?

Reply:
`ov.compile_model` is my preference. Top of the top. Main function in OpenVINO.

Reply:
The "only" problem with `ov.compile_model` is that it is still not aligned with `Core.compile_model`, you cannot pass device name -- don't know why, in my opinion it should be aligned.

The first example that should demonstrate `ov.compile_model` is `ov.compile_model('model.xml')` -- no need to use `read_model` in these particular snippets.

Reply:
> it is still not aligned with `Core.compile_model`, you cannot pass device name

That's of the reasons why most of the changes in this PR won't work.

Reply:
> > it is still not aligned with `Core.compile_model`, you cannot pass device name
> 
> That's one of the reasons why most of the changes in this PR won't work.

You mean changes regarding `compile_model` method, or in general? Ultimately, `ov.compile_model` should be reverted to `core.compile_model` to alleviate this particular issue?

Reply:
@jiwaszki should we add one more overload of `compile_model` with device name? Current one assumes `AUTO` I suppose

Reply:
> @jiwaszki should we add one more overload of `compile_model` with device name? Current one assumes `AUTO` I suppose

This exact way was requested long time ago (ticket ref: 66277 and PR: https://github.com/openvinotoolkit/openvino/pull/8603).

However, It is possible to update the signature with additional `device` argument. It will be just `ov.compile_model(path, device="AUTO")`. It is change in API so it will be safer to do in the next release.

> @slyalin @jiwaszki what is your preference here? and why?

My preference is creation of `Core` manually as it has some other properties available. @ilya-lavrenov "I really forgot that we had compile_model without Core in Python API." -- is the best way to describe my preference.